### ISSUE TITLE
verkko module is missing pandas extension 

### ISSUE THREAD

**Issue by DininduSenanayake** [GenomicsAotearoa/long-read-assembly#1]
<img width="1104" alt="verkko_module_pandaserror" src="https://github.com/GenomicsAotearoa/long-read-assembly/assets/13814535/319ea34e-2b30-43e7-a518-7af491a859cb">

**Comment by DininduSenanayake** [comment]
Fixed now 
```
$ module load verkko/1.3.1-Miniconda3 

$ pip freeze | grep pandas
pandas==1.5.3
```

**Comment by abueg** [comment]
hi @DininduSenanayake ! quick question about the verkko install, did you install the conda version or build from source? just checking as I think the conda version installed might be behind:
```
08:05:52 mahuika01 ~ $ module load verkko
08:08:55 mahuika01 ~ $ verkko --version
bioconda verkko bioconda 1.0
```
vs a version on my uni's HPC:
```
[labueg@login04 /lustre/fs5/vgl/store/labueg/programs]$ conda activate verkko_1.3.1
(verkko_1.3.1) [labueg@login04 /lustre/fs5/vgl/store/labueg/programs]$ which verkko
/vggpfs/fs3/vgl/store/labueg/anaconda3/envs/verkko_1.3.1/bin/verkko
(verkko_1.3.1) [labueg@login04 /lustre/fs5/vgl/store/labueg/programs]$ verkko --version
bioconda verkko bioconda 1.3.1
```

**Comment by DininduSenanayake** [comment]
@abueg  Ah...I did the conda install.  Will try to fix ASAP and get the version 1.3.1 up and running

**Comment by DininduSenanayake** [comment]
@abueg This is fixed now

```
$ module load verkko/1.3.1-Miniconda3

$ verkko --version
bioconda verkko bioconda 1.3.1
```
